mempool.go

```diff
@@ -799,10 +800,9 @@ func (mp *txMemPool) FetchTransaction(txHash *btcwire.ShaHash) (*btcutil.Tx, err
 // more details.
 //
 // This function MUST be called with the mempool lock held (for writes).
-func (mp *txMemPool) maybeAcceptTransaction(tx *btcutil.Tx, isOrphan *bool, isNew, rateLimit bool) error {
-	if isOrphan != nil {
-		*isOrphan = false
-	}
+func (mp *txMemPool) maybeAcceptTransaction(tx *btcutil.Tx,
+	isNew, rateLimit bool) (missingParents []*btcwire.ShaHash, err error) {
```

Comment:
I'll update the patch to not use these so it's consistent with the rest of the btcd code, but really all of the issues we've had with named returns are not with the named return values themselves, but rather using bare return statements (which I did not add).  Named returns without bare return statements only help to document the function's behavior.